data/categories.txt

```diff
@@ -97,7 +97,7 @@ ko:바|술집|맥주|음료|음식
 ja:1居酒屋|バー|パブ|ビール|飲食|飲み屋|呑み屋|酒
 cs:2bar|pivo|pití|jídlo|voda|pitná voda|tekutiny
 sk:2bar|pivo|pitie|jedlo|voda|pitná voda|tekutiny
-nl:2bar|2pub|bier|drinken|eten
+nl:2bar|2pub|2café|bier|drinken|eten
```

Comment:
Because that's how it's translated. A café in Belgium and the Netherlands is a place where they mostly sell beer, and it has nothing to do with coffee. That's also why I switched café with tearoom as the default in the amenity=cafe category. 

See https://nl.wikipedia.org/wiki/Caf%C3%A9 which has "bar" as synonym (kroeg and knijp are also synonyms, but they're a bit older and not used a lot anymore). It also translates to the English page titled "Bar". The terminology changed during history, where first the more expensive ones called themselves "café" (which was a fancy French term in those days), but as cheaper pubs also started to call themselves "café", the more expensive businesses changed their names to even more fancy names like "brasserie" (more focused on selling meals, but still open for drinks only) and when English became popular "tearoom" (focused on being open during the day, and selling coffee and cake).

I even doubted to remove cafe from the amenity=cafe section, but decided to not remove it as mappers also often mis-tag it (thinking the English cafe matches the Dutch café). So by including café in both, the correct results will certainly be in there.

(note that coffeeshop should also be avoided as a translation of café, a coffeeshop is an euphemism for a place where legal soft-drugs are sold in the Netherlands, and it's never used as a place where only coffee is sold.)

I know it's all a bit complicated, but terms evolve as fancier businesses make up their new fancy names, and cheaper businesses copy the fancy names from more expensive ones.